.eslintrc.cjs

```diff
@@ -41,6 +30,7 @@ module.exports = {
       "error",
       { ignores: ["modules"] },
     ],
+    "node/shebang": "off",
```

Comment:
I couldn't find a way to have a shebang in the dist without including it in the source.
I thought about adding to files in the fixup.sh script, but landed on this solution.  I'm open to suggestions for best practice here.